plugins/modules/nxos_route_maps.py

```diff
@@ -332,6 +332,51 @@
                   precedence:
                     description: Set precedence field.
                     type: str
+                  next_hop:
```

Comment:
Could you please add (a) unit tests, (b) an example to show how the newly added key works? Thank you.

Reply:
It`s OK, I found solution.

Reply:
Done. But during tests I found some anomalies in NXOS (ver 9.3(10)). If we enter these commands in switch:
`route-map TEST permit 10`
`set ip next-hop verify-availability 3.3.3.3 track 1 drop-on-fail force-order load-share`
`set ip next-hop verify-availability 5.5.5.5 track 3 drop-on-fail force-order load-share`
then 'show run | sec route-map' produces these output:
`route-map TEST permit 10`
`  set ip next-hop verify-availability 3.3.3.3 track 1`
`  set ip next-hop verify-availability 5.5.5.5 track 3 load-share force-order drop-on-fail`
Provided test and examples does not cover this situation.
From ansible point of view this issue means that in that situation just idempotensy will not work properly.